.changeset/rich-wasps-tell.md

```diff
@@ -0,0 +1,11 @@
+---
+"@meilisearch/instant-meilisearch": minor
+---
+
+- Compatibility with the `Index` widget is added.
+
+## Drawback
+
+Currently, for each index a request is made on, a separated request is made to Meilisearch. For example, if you have two indexes on which to search, two http requests are made.
+
+In the next release of Meilisearch, the `multi-index` route should be released (see [PR](https://github.com/meilisearch/meilisearch/pull/3417)). When it is released, the work-around will be removed and only one request will be done in all cases!
```

Comment:
Again, can we promise that?

Reply:
Pining @gmourier  again hehe

Reply:
It's a multi-search request route if we want to be more precise, and yes, I'm fairly confident we will be able to do deliver it; we already have a working prototype.